src/sound/clip.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+#include "external.hpp"
+
+/// Object to management of buffers
+/// In this version provide management only on one buffer
+class SoundClip {
+private:
+
+	/// Pointer to id of buffers
+	ALuint buffer;
+	/// Path to the audio file
+	std::string path;
+	/// Method to convert channels to mono
+	void convertChannels(int *audio_size, int channels, short** data);

```

Comment:
Make this function take `short* data`
and avoid calling `malloc()`, instead modify the `data` buffer